crates/sui-core/src/authority_active/execution_driver/mod.rs

```diff
@@ -112,31 +113,67 @@ where
                 true
             }
         })
-        .collect();
+        .partition(|(_, (is_sequenced, _))| *is_sequenced);
+
```

Comment:
Could we add a debug logging here to print the number on each side of the partition?

Reply:
good idea